Refactor pages related to garage-sign (#1705)
advancedtelematic/aktualizr: docs/ota-client-guide/modules/ROOT/pages/upload-large-binary.adoc

----
+

. Pull the latest version of the `targets.json` file.
+
[source,bash]
----
garage-sign targets pull \
  --repo <localreponame>
----

. To upload your binary to OTA Connect, specify its name and version.
+

> TIP: You can find the `name` and `version` fields in the `targets.json` file under the `custom` field, as shown in the xref:customise-targets-metadata.adoc#_anatomy_of_targets_json_metadata[example].

Reviewer: ```suggestion
```
Let's get rid of this tip here. It doesn't make a lot of sense, because you can't find the name and version there yet--this is still one step before that metadata is added.

Author: Deleted

----
+

. Pull the latest version of the `targets.json` file.
+
[source,bash]
----
garage-sign targets pull \
  --repo <localreponame>
----

> . Upload your binary to OTA Connect.

Reviewer: You aren't specifying the name and version of the `targets.json` file, you're specifying the name and version of the target you're adding to the `targets.json` file--i.e. the "software version". (Note: this is a place where our terminology choice makes for some awkward sentences. "Specify the name and version of your software version" isn't a good look.)

```suggestion
. Upload your binary to OTA Connect. You'll need to specify the name and version of the binary you're uploading.

```

  --name <name-of-target> \
  --version <target-version>
----
+
TIP: You can find the `name` and `version` fields in the `targets.json` file under the `custom` field, as shown in the xref:customise-targets-metadata.adoc#_anatomy_of_targets_json_metadata[example].

> . If the upload is successful, in the `targets.json` file, specify the binary metadata, as shown in the xref:customise-targets-metadata.adoc#_anatomy_of_targets_json_metadata[example].

Reviewer: Two comments here:

* I know you didn't introduce this, but I think that linking to the example metadata this way might be more distracting than helpful--the command we specify generates the metadata so that the user doesn't have to worry about manually editing `targets.json`. If you think the link is valuable, I'd propose moving it to a footnote.

```suggestion
. If the upload is successful, add it to your local Targets metadata. The name and version you specify here must be the same as the ones you gave in the upload step.

```

Author: Dropped this info and modified the step.
